frontend/src/Service/request.js

```diff
@@ -0,0 +1,18 @@
+async function request(params) {
+  const { url, header, body, method } = params;
+  const headers = {
+    "Content-Type": "application/json",
+    ...header
+  };
+  const reqMethod = method ? "GET" : method;
```

Comment:
Do you need to always get the method "GET"?
if method = 'GET'   reqMethod= 'GET'
if method = 'POST'   reqMethod ='GET'

if you need get  the 'GET' method when params.method is empthy and params.method value when it is not empty  use const reqMethod = !method ? "GET" : method;

Reply:
I saw this issue and will change it to
ReqMethod = method || "GET".
Also this version is temporary, untill we agreed on what data sets we need in various components.

Reply:
It may be better not to merge temporary version ?